py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -32,12 +35,17 @@ def parseArgs(args) -> dict:
         default=100000,
         help='filesize threshold for batch sizes')
 
+    parser.add_argument(
+        '--pretty',
+        dest='pretty',
+        action='store_true',
+        help='print out a pretty summary of the batches')
+
     parser.add_argument(
         '--output',
-        dest='output_dir',
+        dest='output',
         type=str,
-        required=True,
-        help='provide a directory to output')
+        help='filepath to dump out batches as event json list')
```

Comment:
i changed the output format to dump event json out to file and print things out if `--pretty` is defined.